svg_cleaner.py

```diff
@@ -157,8 +172,19 @@ def _clean_elem(self, node):
         elif n.name == 'g':
           if not n.contents:
             continue
+          if 'i:extraneous' in n.attrs:
+            del n.attrs['i:extraneous']
           if not n.attrs and len(n.contents) == 1:
             n = n.contents[0]
+        elif n.name == 'i:pgf' or n.name == 'foreignObject':
+          continue
+        elif n.name =='switch' and len(n.contents) == 1:
+          n = n.contents[0]
+        elif n.name == 'style':
+          # some emacsen don't default 'style' properly, so leave this in.
+          if False and n.attrs.get('type') == 'text/css':
```

Comment:
So, this line (always false if-clause) is kinda  reminder to delete it later when things change, I guess.